docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -95,6 +95,9 @@ Property Name                                        Description
 ``iceberg.rest-catalog.uri``                   REST server API endpoint URI (required).
                                                Example: ``http://iceberg-with-rest:8181``
 
+``iceberg.rest-catalog.warehouse``             Warehouse identifier/location for the catalog (optional).
```

Comment:
in JDBC catalog we call it `iceberg.jdbc-catalog.default-warehouse-dir`
with description "The default warehouse directory to use for JDBC"

the "identifier" is misleading
the config name is unnecessarily different

Reply:
@mosabua do we write "(optional)" for optional properties, or just for required ones (as above), or ?